docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -179,14 +179,69 @@ import { LinkCard } from '@astrojs/starlight/components';
 	<LinkCard title="Composants" href="/fr/guides/components/" />
 </CardGrid>
 
+### Asides
+
+Les encarts (également connues sous le nom de "aside" ou encore "élément en marge") sont utiles pour afficher des informations secondaires à côté du contenu principal d'une page.
+
+Un `<Aside>` peut avoir un `type` optionnel de `note` (par défaut), `tip`, `caution` ou `danger`. La définition d'un attribut `title` remplace le titre par défaut de l'encart.
+
+````mdx
+# src/content/docs/example.mdx
+
+import { Aside } from '@astrojs/starlight/components';
+
+<Aside>Un encart par défaut sans titre personnalisé.</Aside>
+
+<Aside type="caution" title="Attention !">
```

Comment:
```suggestion
<Aside type="caution" title="Prenez garde !">
```

The default `aside.caution` translation in French is `Attention` so we may want to avoid it for a custom title example.